help/destinations/catalog/crm/zendesk.md

```diff
@@ -7,13 +7,13 @@ last-substantial-update: 2023-03-14
 
 [[!DNL Zendesk]](https://www.zendesk.com) is a customer service solution and sales tool.
 
-This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Zendesk] Contacts API](https://developer.zendesk.com/api-reference/sales-crm/resources/contacts/), to create and update identities within a segment as contacts within [!DNL Zendesk].
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Zendesk] Contacts API](https://developer.zendesk.com/api-reference/sales-crm/resources/contacts/), to **create and update identities** within a segment as contacts within [!DNL Zendesk].
```

Comment:
Any reason for bolding "create and update identities"? I think it's clear enough what that does even without bolding it.

Reply:
I can provide context here: for another connector (I forget which one), it wasn't clear from the description what the connector does, so I suggested that we bold the section where we say that the connector creates and updates identities. 

@mohan-h - I see that you have taken the same approach here. I think that's fine and I don't see any harm in bolding this part.